articles/overview/overview.md

```diff
@@ -0,0 +1,67 @@
+
+## What can a quantum computer do?
+
+A quantum computer isn’t a supercomputer that can do everything faster, it just does some select things extremely well. 
+
+### Quantum simulation
+
+Since quantum computers use quantum particles as their processors, they are well-suited for modeling other quantum systems. Photosynthesis, superconductors, and complex molecules are examples of quantum systems that can be simulated using quantum programs.
+
+### Cryptography and Shor’s algorithm
+
+In 1994, Peter Shor showed that a scalable quantum computer can break every widely used encryption technique. Classical cryptography relies on the intractability of operations on large numbers, such as factorization of large numbers into two prime numbers, and quantum computers excel at factorization. 
+
+### Search and Grover’s algorithm
+
+In 1996, Lov Grover developed a quantum algorithm that dramatically sped up the solution to unstructured searches faster than any classical algorithm can do. This increased speed becomes significant in lists that contain millions or billions of items.
+
+### Optimization
+
+Optimization is the task of balancing a system for maximum efficiency and minimum cost. A quantum computer can speed up optimization algorithms and find solutions that otherwise were not possible. In addition to optimizing traffic flow to reduce congestion, there is airplane gate assignment, package delivery, job scheduling and more. With breakthroughs in materials science, there will be new forms of energy, batteries with greater capacity, lighter and stronger materials.
+
+### Machine Learning
+
+A majority of numerical calculations in classical computing consists of solving linear systems of equations. This is especially true in the field of machine learning where most of the computation cost goes into calculating the inverse of huge matrices.
+
+Fortunately, there is a quantum algorithm that allows us to approximate a solution exponentially faster than a classical computer. Reducing computation times in these areas will help address the energy crisis, climate change, food scarcity, and personal and precise medical diagnosis.
+
+### Quantum-inspired computing
+
+Quantum-inspired algorithms use quantum principles for increased speed and accuracy, but are implemented on classical computer systems.
+For example, using algorithms developed by Microsoft’s quantum computing team, researchers at Case Western Reserve University have produced MRI scans that are up to three times faster than prior state-of-the-art approaches, as well as scans that are almost 30 percent more precise in measuring a key identifier of disease.
+
+## Current advancements
+
+Recently, Microsoft’s Quantum Lab located at the University of Sydney made breakthroughs in qubit control technology that allow them to scale beyond the physical limitations of current conventional systems. 
```

Comment:
This feels like an odd pair of advancements to select out of everything Microsoft has done in this area; if we keep this section, I'd suggest workshopping with @cpalmer2020 on this one to find some other appropriate achievements to pull in.

Reply:
Also, "physical limitations of current conventional systems" for what, temperature, precision of qubit manipulation or? Some explanation of what our control system has improved will be helpful.